spring-cloud-dataflow-shell/pom.xml

```diff
@@ -30,6 +30,13 @@
 			<plugin>
 				<groupId>org.springframework.boot</groupId>
 				<artifactId>spring-boot-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<goals>
+							<goal>repackage</goal>
```

Comment:
**[UNRELATED]** this was not being called during `clean install` which resulted in non-executable shell jar.